lib.php

```diff
@@ -86,12 +86,17 @@ function block_carousel_pluginfile($course, $birecordorcm, $context, $filearea,
     send_stored_file($file, null, 0, $forcedownload, $options);
 }
 
-
+/**
+ * The options for the image file picker for carousel slides.
+ *
+ * @return array Options to be passed to add_element().
+ */
 function block_carousel_file_options() {
     global $CFG;
 
     return array(
-        'accepted_types' => array('image', 'web_video'),
+        'accepted_types' => ['ai', 'bmp', 'gif', 'ico', 'jpe', 'jpeg', 'jpg', 'pct', 'pic', 'pict', 'png', 'tif', 'tiff', 'avi',
```

Comment:
This seems like a large list, do we actually want ico, pct, pic, pict, tiff? 

On the video front, can they be collapsed into just 'video' ?

Reply:
ping @Peterburnett for more eyes

Reply:
I wonder if we can narrow down the list of supported images, but keep web_video the same.

Reply:
This looks relatively comprehensive, but why cant we just use web_image, web_video? 
Those are these specfically to cover these aliases iirc

Reply:
> This looks relatively comprehensive, but why cant we just use web_image, web_video? 
> Those are these specfically to cover these aliases iirc

Afaik, it's mainly because svg is not supported (yet?) and some other files probably shouldn't be either like ai ico etc.

Reply:
+1 to just swap image -> web_image, that ticks all the boxes

Reply:
+1 from me as well. Realised just now @Peterburnett had mentioned web_image (not just image)

Reply:
.svg is included in the web_image list.

Reply:
doh, I actually checked this before I commented but my greping failed me, I think just 'gif', 'jpeg', 'jpeg' 'png' and the video

Reply:
done.